TheMovieDb/New Group/Controllers/SearchResultViewControllerç.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SearchResultViewController.swift
+//  TheMovieDb
+//
+//  Created by Angel Trejo on 23/11/21.
+//
+
+import UIKit
+
+final class SearchResultViewController: UIViewController {
+ 
+    var isFiltering: Bool = false
+    var filteredMoviesCount: Int = 0
+    var moviesCount: Int = 0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
```

Comment:
Nitpick: Let's keep the white space lines to a maximum of 1 (SwiftLint should have complained about this)